src/python/WMCore/WorkQueue/WorkQueue.py

```diff
@@ -204,6 +206,15 @@ def __init__(self, logger = None, dbi = None, **params):
                                                               requireBlocksSubscribed = not self.params['ReleaseIncompleteBlocks'],
                                                               fullRefreshInterval = self.params['FullLocationRefreshInterval'],
                                                               updateIntervalCoarseness = self.params['LocationRefreshInterval'])
+        
+        # used for global WQ
+        if self.params.get('ReqMgrServiceURL'):
+            self.reqmgrSvc = ReqMgr(self.params['ReqMgrServiceURL'])
```

Comment:
Sorry for my stupid comment, but why we cannot use RequestDBReader for global WQ as well?
I always confuse these 2 :-(